### PR TITLE
[australia] - cfp extend

### PR DESCRIPTION
**Pull request by swapnilogale**
Post for CFP extension (only if required)

----
📚 Documentation preview 📚: https://writethedocs-www--2193.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs extending CFP by a week

File: docs/conf/australia/2024/news/cfp-extend.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+===========================
+
+Greetings fellow documentarians!
+
+The conference is now just under three months away, and we hope you’re already planning for this year's onsite installment of Write the Docs Australia conference.
+
+New deadline to submit your proposal is 6th Sept
+--------------------------------------------------
+
+We added an extra week to the deadline because there are enough things to worry about this time of the year, and we wanted to give you some more time to prepare your submissions.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/cfp-extend.rst]: > there are enough things to worry about this time of the year

there are? Without context this confuses me

File: docs/conf/australia/2024/news/cfp-extend.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Make sure to get your talk in before the new deadline; we already have a lot of great proposals come in.
+
+Check out the `Call for Proposals <https://www.writethedocs.org/conf/australia/2024/cfp/>`_ page for tips and examples to help inspire you.
+And if you’ve got an idea but aren't sure what to do with it, feel free to ask for feedback on `Slack <http://slack.writethedocs.org/>`_. Your fellow documentarians will be happy to help!
+
+We will send out the decision emails to all who submitted by **mid-September**, and the final talks will be published by **end of September** or **early October**.
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/cfp-extend.rst:22]: Are these changed or unchanged from the original, let's be specific

**Review by plaindocs — APPROVED** [review]
Approval for when you need it

**Pull request merged**